### DISCUSSION TITLE
Keep session with single auth

### DISCUSSION THREAD

**Discussion by nisa10880** [Web3Auth/Web3Auth#1240] [Help]
Hello,
How to keep user logged in when using web3 single auth. The user is disconnected when I refresh the page.

**Comment by nisa10880** [comment]
Nobody else faced this issue?

**Reply by enu-kuro** [reply]
I'm experiencing the same issue.
https://github.com/Web3Auth/web3auth-web/discussions/753#discussioncomment-4610061

**Comment by shahbaz17** [comment]
Hey @nisa10880 

Which version of the SDK are you using? Also, can you test the scenario in [this example app](https://github.com/Web3Auth/examples/tree/main/web-single-factor-auth/react-evm-sfa-example)? and share your findings.

**Reply by nisa10880** [reply]
I tested all versions, including the last version (4.3.2). The example is no longer working anymore for me. I can't see the login button. But even when it was working, the session wasn't kept.

**Reply by shahbaz17** [reply]
Hey @nisa10880 

For many community members and us, it is working. Check out https://github.com/orgs/Web3Auth/discussions/1174#discussioncomment-4858785

Please follow the instructions provided in the above thread.

**Reply by shahbaz17** [reply]
https://github.com/orgs/Web3Auth/discussions/1174

**Reply by nisa10880** [reply]
The problem exists also in the example. Here is a recording to be more clear.

https://user-images.githubusercontent.com/39282363/218816436-4e7c608a-7e04-48c1-aa97-42190af28332.mov

**Reply by nisa10880** [reply]
@shahbaz17 Hello, do you have also the same bug ? The screenshot is from the example repo
